hack/generator/pkg/conversions/testdata/CopyJSONProperty.golden

```diff
@@ -16,7 +16,7 @@ type Person struct {
 func (person *Person) AssignPropertiesFromPerson(source *vNext.Person) error {
 
 	// JSONBlob
-	person.JSONBlob = source.JSONBlob.DeepCopy()
+	person.JSONBlob = *source.JSONBlob.DeepCopy()
```

Comment:
I don't think this compiles.

And, if `person.JSONBlob` was optional, we'd generate this:

``` go
person.JSONBlob = &*source.JSONBlob.DeepCopy()
```
which also won't compile.

Reply:
Added tests to check, and it looks fine.